source/features/comments-time-machine-links.tsx

```diff
@@ -94,7 +94,7 @@ async function showTimemachineBar(): Promise<void | false> {
 		const lastCommitDate = await elementReady([
 			'.repository-content .Box.Box--condensed relative-time',
 			'[itemprop="dateModified"] relative-time' // "Repository refresh" layout
-		].join(), {waitForChildren: false});
+		].join());
```

Comment:
I don’t think this is correct, we only use the attribute, not the children

Reply:
Also if you want to drop all pre-refresh code you can do so in a new PR, as long as no other changes are done in the same PR, because they’d be lost.

Reply:
> I don’t think this is correct, we only use the attribute, not the children

Going to skip this for now as I don't have time to debug.

> Also if you want to drop all pre-refresh code you can do so in a new PR, as long as no other changes are done in the same PR, because they’d be lost.

Sure